common/metrics/tags.go

```diff
@@ -162,6 +163,13 @@ func TaskQueueTypeTag(tqType enumspb.TaskQueueType) Tag {
 	return &tagImpl{key: TaskTypeTagName, value: tqType.String()}
 }
 
+func WorkerBuildIdTag(buildId string) Tag {
+	if buildId == "" {
+		buildId = "_unversioned_"
+	}
+	return &tagImpl{key: workerBuildId, value: buildId}
```

Comment:
we can't do this, too much cardinality

Reply:
let's keep it at least for the pre-release version to help with debugging.